circle.yml

```diff
@@ -12,7 +12,7 @@ dependencies:
   cache_directories:
     - testapp/node_modules
   post:
-    - ./node_modules/.bin/gulp format:enforce
+    - ./node_modules/.bin/gulp lint
```

Comment:
I've added the format enforce into the lint similar to blocking-proxy. I also would rather have the test fail immediately if lint / format has errors.